app/cbt/utils/emacs.py

```diff
@@ -0,0 +1,18 @@
+import sys
+import subprocess
+import typing
+
+# Concatenate strings using an Emacs subprocess as a show of
+# dominance. Only use with known static data to make 100% sure nobody
+# can somehow escape the quotes and run arbitrary code.
+def emacs_concat_string(str1: str, str2: str, sep: str=""):
+    # princ prints to stdout without separators (aka quotes)
+    command = f"emacs --batch --eval '(princ (concat \"{str1}\" \"{sep}\" \"{str2}\"))'"
+    return subprocess.check_output(command, text=True, shell=True, stderr=subprocess.STDOUT).strip()
+
+def emacs_concat_all_the_strings(strings: typing.List[str], sep: str=""):
+    return_val = ""
+    for string in strings:
+        return_val = emacs_concat_string(return_val, string, sep=sep)
+    return return_val
```

Comment:
This seems pretty efficient, please make it less efficient. Wtf do you think we're going for here? Quick code? This is python after all.

Reply:
Future work will improve the inefficiency of the code while maintaining a high coding standard. This will be the best engineered garbage pile of all time.

However, that is outside the scope of this PR.